### ISSUE TITLE
Why discard the logits of the last token?

### ISSUE THREAD

**Issue by Felixvillas** [WeihaoTan/TWOSOME#6]
Hi~

I found you discarded the logits of the last token in the function `get_action_and_value`:

```python
# maybe no need to use it, directly use logits
logits = torch.log_softmax(outputs.logits, dim=-1)

logits = logits[:, :-1, :] ###I dont know why...
```

Thanks~

**Comment by WeihaoTan** [comment]
Thanks for reaching out. If I remember correctly, the last token is the special token without any meaning, as we need to concatenate the observation prompt with the action prompt we need to delete it.
